lib/charms/mongodb/v1/shards_interface.py

```diff
@@ -217,9 +217,6 @@ def add_shards(self, departed_shard_id):
         with MongosConnection(self.charm.mongos_config) as mongo:
             cluster_shards = mongo.get_shard_members()
             relation_shards = self._get_shards_from_relations(departed_shard_id)
-
-            # TODO Future PR, limit number of shards add at a time, based on the number of
```

Comment:
mongodb actually limits the amount of total migrations occuring so there is no need to do this